infra-as-code/bicep/machinelearning.bicep

```diff
@@ -189,12 +182,11 @@ resource aiHub 'Microsoft.MachineLearningServices/workspaces@2024-07-01-preview'
       authType: 'AAD'
       category: 'AzureOpenAI'
       isSharedToAll: true
-      useWorkspaceManagedIdentity: true
-      peRequirement: 'NotRequired'
       sharedUserList: []
       metadata: {
         ApiType: 'Azure'
         ResourceId: openAiAccount.id
+        location: location

```

Comment:
Why is this needed now?

Reply:
same as above, probably it's not required but why not?

Reply:
Because it introduces a change for an error. This should never be in a different location other than its parent, as such, having it adds risk of an accidental change.  Let's revert this one, thanks.  It might actually be RO anyhow.